src/main/java/seedu/financialplanner/FinancialPlanner.java

```diff
@@ -52,7 +49,7 @@ public void run() {
 
     public void save() {
         try {
-            storage.save(cashflowList, FILE_PATH);
+            storage.save(FILE_PATH);
```

Comment:
Maybe file_path can be stored in the storage class?

Reply:
i put file path in parameter for junit testing:D